microcosm_pubsub/registry.py

```diff
@@ -115,6 +115,21 @@ def compute_bound_handlers(self, bindings):
             if handler in bound_components or handler in bound_component_types
         }
 
+        media_type_handlers = {}
```

Comment:
Move the if stream to a helper function and do a `dict` comprehension?

Reply:
Sure.